table/table_test.go

```diff
@@ -97,6 +100,80 @@ func TestTableIterator(t *testing.T) {
 	}
 }
 
+func TestHashIndexTS(t *testing.T) {
+	b := NewTableBuilder(0, defaultBuilderOpt)
+	defer b.Close()
+
+	filename := fmt.Sprintf("%s%s%d.sst", os.TempDir(), string(os.PathSeparator), rand.Int63())
+	f, err := y.OpenSyncedFile(filename, true)
+	if t != nil {
+		require.NoError(t, err)
+	} else {
+		y.Check(err)
+	}
+	keys := [][]byte{
+		y.KeyWithTs([]byte("key"), 9),
+		y.KeyWithTs([]byte("key"), 7),
+		y.KeyWithTs([]byte("key"), 5),
+		y.KeyWithTs([]byte("key"), 3),
+		y.KeyWithTs([]byte("key"), 1),
+	}
+	for _, k := range keys {
+		b.Add(k, y.ValueStruct{Value: k, Meta: 'A', UserMeta: 0})
+	}
+	f.Write(b.Finish())
+	f.Close()
+	f, _ = y.OpenSyncedFile(filename, true)
+	table, err := OpenTable(f, options.MemoryMap)
+
+	it, ok := table.PointGet(y.KeyWithTs([]byte("key"), 10))
+	require.True(t, ok)
```

Comment:
Could you use Assert to remove the dependency of "require" library?

Reply:
`require` is used in every test. Maybe fix them altogether in next PR?

Reply:
OK